rules/dead-doc-block/src/Rector/Node/RemoveNonExistingVarAnnotationRector.php

```diff
@@ -118,8 +120,16 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        $phpDocInfo->removeByType(VarTagValueNode::class);
+        $comments = $node->getComments();
+        if (isset($comments[1]) && $comments[1] instanceof Comment) {
+            $node->setAttribute(AttributeKey::COMMENTS, null);
+            $node->setDocComment(new Doc($comments[1]->getText()));
+            $node->setAttribute(AttributeKey::PHP_DOC_INFO, null);
```

Comment:
This will be probably needed in more generic way in one place, where comments are removed.

Reply:
need a Util class to rollback comment?

Reply:
Something like that. We'll see what else is reported